.gitignore

```diff
@@ -2,4 +2,13 @@ leelaz
 *.o
 *.d
 training/tf/venv
-weights.txt
\ No newline at end of file
+weights.txt
+.idea
+/msvc/packages
+/msvc/VS2015/x64/Debug
+/msvc/VS2015/x64/Release
+/msvc/x64/Debug
+/msvc/x64/Release
+/msvc/leela-zero2015.VC.db
+/msvc/leela-zero2015.VC.VC.opendb
+/msvc/.vs/leela-zero2015/v14/.suo
```

Comment:
These last 3 look unfamiliar to me.

Reply:
I was going to remove them from .gitignore, but then I did not see how to exclude them from the commit using VS. I removed them and pushed from the cmd line.